data/accounts/pt_PT/acctchrt_brokerage.gnucash-xea

```diff
@@ -1,48 +1,20 @@
-  xmlns:cust="http://www.gnucash.org/XML/cust"
-  xmlns:employee="http://www.gnucash.org/XML/employee"
-  xmlns:entry="http://www.gnucash.org/XML/entry"
-  xmlns:fs="http://www.gnucash.org/XML/fs"
-  xmlns:gnc="http://www.gnucash.org/XML/gnc"
-  xmlns:gnc-act="http://www.gnucash.org/XML/gnc-act"
-  xmlns:invoice="http://www.gnucash.org/XML/invoice"
-  xmlns:job="http://www.gnucash.org/XML/job"
-  xmlns:lot="http://www.gnucash.org/XML/lot"
-  xmlns:order="http://www.gnucash.org/XML/order"
-  xmlns:owner="http://www.gnucash.org/XML/owner"
-  xmlns:price="http://www.gnucash.org/XML/price"
-  xmlns:recurrence="http://www.gnucash.org/XML/recurrence"
-  xmlns:slot="http://www.gnucash.org/XML/slot"
-  xmlns:split="http://www.gnucash.org/XML/split"
-  xmlns:sx="http://www.gnucash.org/XML/sx"
-  xmlns:taxtable="http://www.gnucash.org/XML/taxtable"
-  xmlns:trn="http://www.gnucash.org/XML/trn"
-  xmlns:ts="http://www.gnucash.org/XML/ts"
-  xmlns:tte="http://www.gnucash.org/XML/tte"
-  xmlns:vendor="http://www.gnucash.org/XML/vendor">
+<gnc-account-example xmlns="http://www.gnucash.org/XML/" xmlns:act="http://www.gnucash.org/XML/act" xmlns:addr="http://www.gnucash.org/XML/addr" xmlns:bgt="http://www.gnucash.org/XML/bgt" xmlns:billterm="http://www.gnucash.org/XML/billterm" xmlns:book="http://www.gnucash.org/XML/book" xmlns:bt-days="http://www.gnucash.org/XML/bt-days" xmlns:bt-prox="http://www.gnucash.org/XML/bt-prox" xmlns:cd="http://www.gnucash.org/XML/cd" xmlns:cmdty="http://www.gnucash.org/XML/cmdty" xmlns:cust="http://www.gnucash.org/XML/cust" xmlns:employee="http://www.gnucash.org/XML/employee" xmlns:entry="http://www.gnucash.org/XML/entry" xmlns:fs="http://www.gnucash.org/XML/fs" xmlns:gnc="http://www.gnucash.org/XML/gnc" xmlns:gnc-act="http://www.gnucash.org/XML/gnc-act" xmlns:invoice="http://www.gnucash.org/XML/invoice" xmlns:job="http://www.gnucash.org/XML/job" xmlns:lot="http://www.gnucash.org/XML/lot" xmlns:order="http://www.gnucash.org/XML/order" xmlns:owner="http://www.gnucash.org/XML/owner" xmlns:price="http://www.gnucash.org/XML/price" xmlns:recurrence="http://www.gnucash.org/XML/recurrence" xmlns:slot="http://www.gnucash.org/XML/slot" xmlns:split="http://www.gnucash.org/XML/split" xmlns:sx="http://www.gnucash.org/XML/sx" xmlns:taxtable="http://www.gnucash.org/XML/taxtable" xmlns:trn="http://www.gnucash.org/XML/trn" xmlns:ts="http://www.gnucash.org/XML/ts" xmlns:tte="http://www.gnucash.org/XML/tte" xmlns:vendor="http://www.gnucash.org/XML/vendor">
     <gnc-act:title>
-     Conta de Investimentos
+     Conta de investimentos
     </gnc-act:title>
     <gnc-act:short-description>
-      Conta de corretagem com contas de investimentos relacionados
-        (acções, obrigações, fundos de investimento, fundos de índices, juros, dividendos)
+      Conta de correctagem com contas de investimentos relacionados
+        (acções, obrigações, fundos de investimento, fundos de índices, 
+        juros, dividendos)
```

Comment:
The short description is currently not visible. I made a note about it in https://bugs.gnucash.org/show_bug.cgi?id=745353.

But in general it should fit in a half screen line. Move the rest into the long description.